packages/patternlab/styleguide/source/_patterns/02-molecules/banner-credit.twig

```diff
@@ -1,10 +1,11 @@
-<div class="ma__banner-credit">
-  <div class="ma__banner-credit__container">
-    <div class="ma__banner-credit__icon">
+<div class="ma__banner-credit" aria-hidden="true">

```

Comment:
```suggestion
<div class="ma__banner-credit" aria-hidden="true">
```

can we just add a boolean variable for aria-hidden? I think that will make it make more sense